src/dmd/irstate.d

```diff
@@ -1,5 +1,7 @@
 /**
- * The state of Intermediate Representation (IR), which is what the back-end uses.
+ * Used to help transform statement AST into flow graph.
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
```

Comment:
Can we please not do that ? This boilerplate was removed in a previous PR because it doesn't add anything to the generated documentation.

Reply:
It's a convenience link for the person editing the file. I don't agree with the rationale of removing it, but I'll remove it anyway.

Reply:
I get the rationale for the other links (source, documentation, coverage), but `dlang.org` ?
Anyway, thanks for taking the feedback into account.